Automattic/jetpack
# PHPCS: SAL (part 2)

This PR partially fixes the issue at #21967

#### Changes proposed in this Pull Request:

* This PR updates the SAL files to meet PHPCS standards:
   * "projects/plugins/jetpack/sal/class.json-api-post-base.php"
   * "projects/plugins/jetpack/sal/class.json-api-post-jetpack.php"

#### Jetpack product discussion

n/a

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* For each file, test features impacted by that file. For `class.json-api-post-base.php` for example, the experience of loading posts in the WordPress.com dashboard will be broken if there is unexpected output in the file in some places.
* Also, you can use the WordPress.com developer console to test an endpoint such as the WP.COM API GET request for `/sites/{blog_id}/posts/{post_id}` - for a sandboxed WPcom site and a sandboxed Jetpack site. A breaking change will affect the output, whereas it should work fine otherwise.
* We're primarily checking for any broken experiences or PHPCS errors.
